scripts/getNonResonant_gen_mhh_vs_costhetastar.py

```diff
@@ -0,0 +1,118 @@
+#! /bin/env python
+
+from __future__ import division
+
+import math
+
+import ROOT
+from DataFormats.FWLite import Events, Handle
+
+ROOT.gROOT.SetBatch() 
+
+datasets = []
+for i in range(2, 14):
+    datasets.append('/GluGluToHHTo2B2VTo2L2Nu_node_%d_13TeV-madgraph/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM' % i)
```

Comment:
Indeed, but since we don't reweight these two, I thought it's be better to keep them separated from the rest.